tests/TestCase/Mode/CallbackTest.php

```diff
@@ -37,6 +37,7 @@ public function tearDown()
     public function testMaintenanceModeCallback()
     {
         Configure::write('Wrench.enable', true);
+        Configure::write('Wrench.whitelist', ['127.0.0.1']);
```

Comment:
Why adding this everywhere ? A better way to test this would be to create a new test, disabling the mode with `Configure::write('Wrench.enable', false);` and only providing the whitelist to see if the response is matching.